src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam.
+-/
+import analysis.special_functions.trigonometric
+import data.real.basic
+import analysis.special_functions.pow
```

Comment:
Do you need this import? I don't see `rpow` or `cpow` being used anywhere.

Reply:
Nope, I don't, removed.